zipline/api/android/zipline.api

```diff
@@ -202,7 +202,6 @@ public abstract interface class app/cash/zipline/ZiplineFunction {
 
 public final class app/cash/zipline/ZiplineManifest {
 	public static final field Companion Lapp/cash/zipline/ZiplineManifest$Companion;
-	public synthetic fun <init> (ILapp/cash/zipline/ZiplineManifest$Unsigned;Ljava/util/Map;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Ljava/util/Map;Lkotlinx/serialization/internal/SerializationConstructorMarker;)V
```

Comment:
Unclear to me what the binary-compatibility consequences are of dropping these synthetic constructors. It would be pretty horrible if code compiled for 1.9.10 crashed when deserializing these!

But I don’t think I have many choices here; this is a deep implementation detail of kotlinx.serialization.

Reply:
Phew, looks like it’s safe!
https://github.com/Kotlin/kotlinx.serialization/issues/2209